packages/lumx-core/src/scss/components/text-field/_index.scss

```diff
@@ -187,7 +188,7 @@
 
 // Hover state
 .#{$lumx-base-prefix}-text-field--theme-light {
-    .#{$lumx-base-prefix}-text-field__wrapper:hover {
+    .#{$lumx-base-prefix}-text-field__wrapper:hover:enabled {
```

Comment:
Same here `:enabled` does not work in `<div>`

Reply:
Should be fixed